_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -64,6 +56,12 @@ If the [explicit semantic role][] on the target element is incorrectly used, and
 - [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
 - [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
 
+HTML elements with an [implicit semantic role][] corresponding with [explicit semantic role][] are not tested in this rule because some of these elements have additional requirements. For example, an element that has an implicit semantic role that is identical to its explicit semantic role is a `ul` element that has `role="list"`. However, there is no native HTML element to "group" `li` elements in a `ul` element.
```

Comment:
I may be wrong here, but the below notes are reasoning why some things are excluded by this rule.

I do not think that belongs in the background section?

Reply:
I've followed the pattern for PR #1342 where the information of HTML elements within `iframe` not being assessed is moved to the background section.  It's also giving information on why the rule has been written this way and not another- [comment](https://github.com/act-rules/act-rules.github.io/issues/1305#issuecomment-644622078) by @Jym77 .

Happy to change if you still feel like it should stay in the Applicability section.